lib/engine/game/g_1828/step/merger.rb

```diff
@@ -330,12 +330,10 @@ def identify_odd_share(entity)
             if @player_selection
               @odd_share = @merger.name.include?(@player_selection) ? merger_share : target_share
               @player_selection = nil
-            elsif !merger_share
+            elsif !merger_share || entity.num_shares_of(@merger) <= num_system_shares
```

Comment:
@crericha does changing the order that these things are checked have a possible impact?

Three conditionals are checked:
* A) `!merger_share`
* B) `!target_share`
* C) `entity.num_shares_of(@merger) <= num_system_shares`

```
elsif A
  ...
elsif B
  ...
eslif C
  ...
```

is slightly different from
```
elsif A || C
  ...
elsif B
  ...
```

With the old code, `!A && B && !C` and `!A && B && C` both lead to `@odd_share = merger_share` (ie, with `!A && B`, `C` doesn't matter)

With the new code, `!A && B && C` leads to `@odd_share = target_share`

In practice with how 1828 works, I don't know if this actually changes anything. All the games validate so it's probably fine?

Reply:
The two cases can be combined. Ship it. 👍

Reply:
Thanks @crericha !